WooCommerce/Classes/ViewModels/Order Details/Aggregate Order Items/AggregateOrderItemCellViewModel.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+import Yosemite
+
+
+// MARK: - View Model for an aggregate order item cell
+//
+struct AggregateOrderItemCellViewModel {
```

Comment:
from the diffs in `ProductDetailsTableViewCell`, it looks like this struct only has to expose `imageURL`, `name`, `quantity`, `price`, and `sku`. could you make sure the other properties are private?

Reply:
after taking a pass of the code, these 3 view models feel like rendering the same fields:
- `AggregateOrderItemCellViewModel`
- `RefundedOrderItemCellViewModel` 
- `RefundedProductCellViewModel`

even though these three view models derive the view model fields from different data models, if the fields are the same (look like they all expose `imageURL`, `name`, `quantity`, `price`, and `sku`), I'd suggest making one cell view model and implement different initializers for each data model. Otherwise, if we ever update the price formatting or SKU strings, we'd have to remember updating all 3 places.

I might have missed some other substantial differences between these 3 view models, and feel free to ignore this comment if that's the case!